From cf43eb161ab254100f938fd012bf9a48d392fcfb Mon Sep 17 00:00:00 2001
From: Alynx Zhou <alynx.zhou@gmail.com>
Date: Fri, 16 Aug 2024 16:09:08 +0800
Subject: [PATCH] Add http and https MIME types in desktop entry

This makes it selectable in default app selector in GNOME Settings.
---
 data/re.sonny.Junction.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/re.sonny.Junction.desktop b/data/re.sonny.Junction.desktop
index ca34c46..0fa8109 100644
--- a/data/re.sonny.Junction.desktop
+++ b/data/re.sonny.Junction.desktop
@@ -3,7 +3,7 @@
 Name=Junction
 Comment=Application chooser
 Exec=re.sonny.Junction %U
-MimeType=x-scheme-handler/x-junction
+MimeType=x-scheme-handler/x-junction;x-scheme-handler/https;x-scheme-handler/http
 Terminal=false
 Type=Application
 Categories=Settings;System;Utility;GNOME;GTK;
